pkg/autodiscovery/listeners/ecs.go

```diff
@@ -130,6 +130,11 @@ func (l *ECSListener) refreshServices(firstRun bool) {
 	}
 
 	for _, c := range meta.Containers {
+		// Skip containers for which ECS failed to retrieve metadata
+		if c.DockerID == "" {
+			log.Info("Skipping a container for which ECS is reporting an empty ID: name %q, docker name: %q, image %q, image id: %q", c.Name, c.DockerName, c.Image, c.ImageID)
```

Comment:
could this become spammy? this method is called continuously so I'm thinking it could be better to log in debug here.
In any case, the issue should be is transient and the next time we call the method we will most likely receive a complete payload from the ecs api, correct?

Reply:
Agree, if we know it could be a "normal" case, it should be a `debug` or  a `trace` log.

Reply:
Log changed from `Info` to `Debug`: https://github.com/DataDog/datadog-agent/pull/6938/commits/0b6ff584507b79db308cc7d2ef57b5771291160d 👍🏻